renderer/pages/main.tsx

```diff
@@ -1,113 +1,194 @@
-import { Button, Checkbox, Divider, Flex, FormControl, FormLabel, Input, Table, TableCaption, TableContainer, Tbody, Td, Text, Th, Thead, Tr } from "@chakra-ui/react"
-import { useEffect, useState } from "react"
-import BaseLayout from "../components/BaseLayout"
-import electron from "electron"
-import _ from "lodash"
+import {
+  Button,
+  Checkbox,
+  Divider,
+  Flex,
+  FormControl,
+  FormLabel,
+  Heading,
+  Input,
+  Table,
+  TableCaption,
+  TableContainer,
+  Tbody,
+  Td,
+  Text,
+  Th,
+  Thead,
+  Tr,
+} from "@chakra-ui/react";
+import { useEffect, useState } from "react";
+import BaseLayout from "../components/BaseLayout";
+import electron, { clipboard } from "electron";
+import _ from "lodash";
 
-export const PasswordView = ({ name, value, setPassList }) => {
-    const [show, setShow] = useState(false)
+export const PasswordView = ({ label, name, value, setPassList }) => {
+  const [show, setShow] = useState(false);
 
-    const handleDelete = async() => {
-        electron.ipcRenderer.send('delete-pass', name)
-        const passwords = await electron.ipcRenderer.invoke('pass-request')
-        setPassList(passwords)
-    }
+  const handleDelete = async () => {
+    electron.ipcRenderer.send("delete-pass", name);
+    const passwords = await electron.ipcRenderer.invoke("pass-request");
+    setPassList(passwords);
+  };
 
-    return (
-        <Tr>
-            <Td>{name}</Td>
-            <Td>{show ? value : <em>Hidden</em>}</Td>
-            <Td>
-                <Button mr={2} colorScheme="orange" onClick={() => setShow(!show)}>Toggle Show</Button>
-                <Button colorScheme="red" onClick={handleDelete}>Delete</Button>
-            </Td>
-        </Tr>
-    )
-}
+  return (
+    <Tr>
+      <Td>{label}</Td>
+      <Td>{name}</Td>
+      <Td>{show ? value : <em>Hidden</em>}</Td>
+      <Td>
+        <Button mr={2} colorScheme="orange" onClick={() => setShow(!show)}>
+          Toggle Show
+        </Button>
+        <Button mr={2} colorScheme="red" onClick={handleDelete}>
+          Delete
+        </Button>
+        <Button colorScheme="green" onClick={() => clipboard.writeText(value)}>
```

Comment:
Change clipboard functionality to use [useClipboard](https://chakra-ui.com/docs/styled-system/utility-hooks/use-clipboard) instead.
```tsx
function Example() {
  const [value, setValue] = React.useState('Hello world')
  const { hasCopied, onCopy } = useClipboard(value)

  return (
    <>
      <Flex mb={2}>
        <Input value={value} isReadOnly placeholder='Welcome' />
        <Button onClick={onCopy} ml={2}>
          {hasCopied ? 'Copied' : 'Copy'}
        </Button>
      </Flex>
      <Editable placeholder='Paste here'>
        <EditablePreview width='100%' />
        <EditableInput />
      </Editable>
    </>
  )
}
```
This way, you can change the button's text to "Copied" so the users know that the password is on their clipboard.